### ISSUE TITLE
Restore subscription when re-install app

### ISSUE THREAD

**Issue by ademariasen** [j3k0/cordova-plugin-purchase#429]
Hi,

I don't found how to restore a purchase after you have deleted the application and it reinstall.
I try with iOS, if a buy a subscription and I remove application, when I reinstall I don't found any function that permit to restore the paid subscription.

how can I do ?

Thank you so much
Andrea

**Comment by remisture** [comment]
https://github.com/Fovea/cordova-plugin-purchase-demo/blob/master/www/js/index.js#L242

`// When store is ready, activate the "refresh" button;
    store.ready(function() {
        var el = document.getElementById('refresh-button');
        if (el) {
            el.style.display = 'block';
            el.onclick = function(ev) {
                store.refresh();
            };
        }
    });`

**Comment by ademariasen** [comment]
Hi remisture,

thank you for your answer. 
But I don't understand of that resolve my problem. 
I think that isn't possibile to restore a purchase without pass the apple information, like a transaction id.
With this procedure I think your refresh the status after your buy a subscription, but not when you re-install the application.
I use an work arround but I don't know if is a good solution.

I found 3 localStorage keys that contain informations about store transaction. I save this in my server and when I reinstall the app I restore those 3 keys.

I tried it and it seems to work but I have yet to do thorough testing.

Thank you
Andrea
